### DISCUSSION TITLE
Question about Chai Terms of Service

### DISCUSSION THREAD

**Discussion by kirmedvedev** [chaidiscovery/chai-lab#248] [Q&A] [answered]
Hello,

Chai Web Application Terms of Service (https://www.chaidiscovery.com/terms-of-service) states that "You further agree that you will not: (i) use, or permit others to use, the Services or any Output or Derived Materials in connection with any benchmarking or other comparison of the Services with any other services or technology".
1. Does this apply only to the web application, or does it also include Chai installed on any HPC cluster?
2. I am using Chai installed on HPC cluster. Will I violate this rule if I include in the paper a protein structure with ligands docked using Chai and any other software?
3. Will I violate this rule if I include graph with any parameters (let's say RMSD) for ligands docked by Chai and any other software?

Thank you!

**Comment by joshim5** [comment, marked as answer]
We are very supportive of community benchmarking efforts.

1. No, you are welcome to run benchmarking experiments using the Chai-lab repository.
2. No, you are welcome to do this.
3. No, you are welcome to do this.

We look forward to seeing your results!

**Reply by kirmedvedev** [reply]
Thank you so much for your reply!
